mlir-tensorrt/compiler/lib/Conversion/TensorRTRuntimeToExecutor/TensorRTRuntimeToExecutor.cpp

```diff
@@ -379,12 +379,9 @@ struct ConvertEnqueueAllocToCall
 
     // Create output memrefs from output descriptors
     SmallVector<Value> results;
+    unsigned offset = 1; // Skip num of results
```

Comment:
Can you give a better description of what this is for? Is this variable even used? should it be `const`?

Reply:
Sure. I can rename to `outputDesOffset`. It basically gets incremented in the code below. I found this issue while trying to debug a Tripy test where function has multiple returns values with different rank.